### ISSUE TITLE
Blue on blue is not very visible for the "Forgot passord" button

### ISSUE THREAD

**Issue by Nilzor** [schibsted/account-sdk-android#354]
![image](https://user-images.githubusercontent.com/990654/46670715-3f93f800-cbd3-11e8-92e5-749b3f182cc5.png)

**Comment by Astach** [comment]
The background of this Forgot Password is supposed to be transparent and is not supposed to be customized and we don't provide a way to do it (afai remember), that's the style of "Secondary action" button.
This is done that way because a secondary action shouldn't catch user attention as much as a primary action. Modifying this behaviour is more than a design customization because it breaks the UI/UX code/identity/guideline/meaning whateveryoumaycallit. It is something that should be discussed with PM and UX peeps
How did you end up with that result?

**Comment by kamilzych** [comment]
@Astach apparently setting a `<item name="colorAccent">` in the client-app breaks the style of buttons in the SDK. This is what I can see after setting colorAccent to red (#f00)

![image](https://user-images.githubusercontent.com/928633/47914475-20217f80-dea0-11e8-92d8-8a7ccc7b2ec6.png)

**Comment by Nilzor** [comment]
I've looked a bit more into this and found a workaround, but it's not optimal:

You do have some kind of customization options for colors documented [here](https://schibsted.github.io/account-sdk-android/ui/#customize-colors). By setting `schacc_primaryEnabled` to `#CCC` I'm getting a gray-ish text which is visible on the blue background also for the "Forgot password" button. The problem with the workaround is that that color is also used for the password `TextView` border color when focused. Why? 

![image](https://user-images.githubusercontent.com/990654/48066441-e4532680-e1cd-11e8-96d9-e9cd31aa94db.png)

tl;dr I think you should clean up the color override options a bit. Color customization is nice for branding, but when it doesn't work and even the default is broken then we have a problem. First make the default work even when `colorAccent` and `colorPrimary` is set in the apps (which they will be)

**Comment by Nilzor** [comment]
We, and other clients, can work fix this by forcing Theme on the password activity:

```
<activity                
  android:name="com.schibsted.account.ui.login.flow.password.PasswordActivity"
  android:theme="@style/OurTheme" />
```

and then in the style file:
```
<resources xmlns:tools="http://schemas.android.com/tools">
    <style name="OurTheme" parent="@style/Theme.AppCompat.Light">
        <item name="colorPrimary">@color/base_color_primary_1</item>
        <item name="colorPrimaryDark">@color/base_color_primary_4</item>
        <item name="colorAccent">@color/base_color_accent_1</item>
        <item name="android:statusBarColor">@color/base_color_status_bar</item>
        <item name="android:windowLightStatusBar" tools:ignore="NewApi">@bool/enableWindowLightStatusBar</item>
    </style>
</resources> 
```
